doc/gallery/plot_proportional_colorbar.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""
+============================================
+Proportional colorbar for discrete colormaps
+============================================
+
+Use ``cbar_kwargs`` keyword to specify the number of ticks.
+``spacing`` can be used to draw the ticks proportional too.
+"""
+from __future__ import absolute_import
```

Comment:
Sorry, please ignore our bot on this one: remove this line

Reply:
Yes, the bot has been terminated -- if you merge in master, it should no longer bother you!